integration/shareddata/composites.go

```diff
@@ -40,6 +40,9 @@ var Composites = &Values[string]{
 		"array-three-reverse": bson.A{nil, "foo", int32(42)},
 		"array-empty":         bson.A{},
 		"array-null":          bson.A{nil},
+		"array-numbers-asc":   bson.A{int32(42), int32(43), int32(45)},
+		"array-strings-desc":  bson.A{"c", "b", "a"},
+		"array-composite":     bson.A{nil, primitive.Timestamp{T: 42, I: 13}, false, "z", primitive.NilObjectID},
```

Comment:
I wanted to add more concrete one but it failed on many tests so for now I created this issue https://github.com/FerretDB/FerretDB/issues/1607